e2e/jest.e2e.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  rootDir: './',
+  preset: 'jest-puppeteer',
+  testMatch: ['<rootDir>/*.test.js'],
+  transformIgnorePatterns: ['node_modules/(?!variables/.*)']
```

Comment:
why do we have to ignore everything but the `variables` folder in `node_modules`?

Reply:
Hmm, when I originally updated jest to latest, this solved an issue I kept encountering, but I just tested without it (looking to copy the error message) and now it works. 🤷‍♀️ I removed transformIgnorePatterns from both jest configs in the latest.